### PR TITLE
Initialise script engine off main thread

### PR DESCRIPTION
**Pull request by AaronVanGeffen**

### CODE REVIEW

File: src/openrct2/Context.cpp [hunk 1]
```diff
@@ -574,6 +570,18 @@ namespace OpenRCT2
             OpenProgress(STR_LOADING_GENERIC);
         }
 
+        void InitialiseScriptEngine()
+        {
+#ifdef ENABLE_SCRIPTING
+            // TODO: add dedicated string
+            OpenProgress(STR_LOADING_GENERIC);
```
Comment (AaronVanGeffen) [inline src/openrct2/Context.cpp:577]: What would be a good title for this? Just 'Initialising script engine...'?

Comment (Basssiiie) [reply src/openrct2/Context.cpp:577]: Maybe "Loading plugins..." would make more sense for non-developers?

Comment (AaronVanGeffen) [reply src/openrct2/Context.cpp:577]: Hmm, the engine gets initialised even if you don't have any plugins installed, though. Perhaps 'Initialising plugin engine...' then?

Comment (Basssiiie) [reply src/openrct2/Context.cpp:577]: "Checking for plugins..." ? 😆

**Review by Basssiiie — APPROVED** [review]
Tested it and it looks good to me! Plugins now don't load at all anymore during the preloader scene, which is good. Nice work! 😄

**Pull request merged**